opentelemetry-sdk/Cargo.toml

```diff
@@ -48,7 +48,8 @@ jaeger_remote_sampler = ["trace", "opentelemetry-http", "http", "serde", "serde_
 logs = ["opentelemetry_api/logs", "crossbeam-channel", "async-trait", "serde_json"]
 logs_level_enabled = ["logs", "opentelemetry_api/logs_level_enabled"]
 metrics = ["opentelemetry_api/metrics", "regex", "async-trait"]
-testing = ["opentelemetry_api/testing", "trace", "metrics", "logs", "rt-async-std", "rt-tokio", "rt-tokio-current-thread", "tokio/macros", "tokio/rt-multi-thread"]
+span = ["opentelemetry_api/trace"]
```

Comment:
A separate 'span' feature is unnecessary, given that we already have a 'trace' feature.